api/indigo.h

```diff
@@ -410,6 +410,12 @@ CEXPORT int indigoCheckQuery(int item);
 // Returns one if structure contains RGroup features (RSites, RGroups or attachment points
 CEXPORT int indigoCheckRGroups(int item);
 
+// Returns check result for Indigo object as text file for requested properties as JSON
+CEXPORT const char* indigoCheck2(const char* item, const char* check_flags, const char* load_params);
```

Comment:
Better avoid adding new functions those return `const char *` (and gradually kill the old ones) since this pattern provokes to return a pointer to a temporary automatic variable, which lifetime ends with return.